frontend/lib/onboarding/onboarding-thanks.tsx

```diff
@@ -7,7 +7,8 @@ export const OnboardingThanks: React.FC<{ next: string }> = ({ next }) => {
     <Page title="Thanks for signing up!" className="content has-text-centered">
       <h1>Thanks for signing up!</h1>
       <p className="subtitle is-4 is-marginless">
-        An email to verify your account is on its way.
+        An email to verify your account is on its way, from{" "}
+        <strong>no-reply@justfix.nyc</strong>.
```

Comment:
This is actually an environment variable I think, or at the very least it's something set in the back-end, so it'd be nice to pull from that just in case it ever changes... for now you can just file an issue to address it later tho!

Reply:
Noice! ok sounds good